src/components/Choice/src/Choice.vue

```diff
@@ -47,15 +49,43 @@ export default {
 			default: 'single-select',
 			validator: (modeValue) => ['single-select', 'multi-select'].includes(modeValue),
 		},
+		/**
+		 * Background color of a selected option
+		 */
+		selectedColor: {
+			type: String,
+			default: '#222',
+			validator: (color) => chroma.valid(color),
+		},
 	},
 
 	data() {
 		return {
 			currentValue: this.selected,
 			isMultiSelect: this.mode === 'multi-select',
+			getSelectedStyle: () => ({
```

Comment:
i don't think you need to do this... CSS variables will cascade naturally, just set them on the parent and the child ChoiceOption component will inherit them, or is there a particular reason you chose to do it this way?